recipes/labjackpython/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "LabJackPython" %}
+{% set org = "labjack" %}
+{% set upstreamversion = "4-24-2014" %}
+{% set version = "20140424" %}
+{% set sha256 = "9cf7a6fca9f1308b60a4442dd1410af216fb1d38e49aa5c1ca1e670958c4bcf5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    fn: {{ name|lower }}-{{version}}-{{ sha256 }}.zip
+    url: https://github.com/{{ org }}/{{ name }}/archive/{{ upstreamversion }}.zip
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: True  # [py3k]
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+test:
+  imports:
+    - u3
+    - u6
+    - ue9
+
+about:
+  home: http://labjack.com/support/labjackpython
+  summary: "Python module for communicating with the LabJack U3/U6/UE9/U12."
+  license: MIT X-11
+  license_family: MIT
```

Comment:
Please raise an issue upstream and ask for a proper license file. This makes it easier for people to determine how the code is licensed. Also please ask that it gets included in the `MANIFEST.in`. This will help with packaging the license file when the project starts making PyPI releases.